diff --git a/project_cust_38/Cust_config.py b/project_cust_38/Cust_config.py
index 42815a8..41d6de5 100644
--- a/project_cust_38/Cust_config.py
+++ b/project_cust_38/Cust_config.py
@@ -138,9 +138,14 @@ class BaseConfig(metaclass=SingletonMeta):
     def _get_horizontal(self, name=None):
         select = name if name else '*'
         where = f" WHERE {self._key_col} = ?"
+        pg_where = f" WHERE {self._key_col} = %s"
         count_articles = CSQ.custom_request_c(
             self._CONFIG_DB,
-            f'SELECT COUNT(*) as cnt FROM {self.__table__} {where}',
+            CSQ.SqlQuery(
+                sqlite=f'SELECT COUNT(*) as cnt FROM {self.__table__} {where}',
+                postgres=f'SELECT COUNT(*) as cnt FROM {self.__table__} {pg_where}',
+                canBranch=self.__table__ == 'app_config',
+            ),
             rez_dict=True, one=True,
             list_of_lists_c=[self._val_col]
         )
@@ -150,7 +155,11 @@ class BaseConfig(metaclass=SingletonMeta):
             return
         result = CSQ.custom_request_c(
             self._CONFIG_DB,
-            f'SELECT {select} FROM {self.__table__} {where}',
+            CSQ.SqlQuery(
+                sqlite=f'SELECT {select} FROM {self.__table__} {where}',
+                postgres=f'SELECT {select} FROM {self.__table__} {pg_where}',
+                canBranch=self.__table__ == 'app_config',
+            ),
             rez_dict=True,
             one=True,
             list_of_lists_c=[self._val_col]
@@ -165,10 +174,16 @@ class BaseConfig(metaclass=SingletonMeta):
 
     def _get_vertical(self, name=None):
         where, params = '', [[]]
+        pg_where = ''
         if name:
             where = f'WHERE {self._key_col} = ?'
+            pg_where = f'WHERE {self._key_col} = %s'
             params = [name]
-        query = f'SELECT * FROM {self.__table__} {where}'
+        query = CSQ.SqlQuery(
+            sqlite=f'SELECT * FROM {self.__table__} {where}',
+            postgres=f'SELECT * FROM {self.__table__} {pg_where}',
+            canBranch=self.__table__ == 'config',
+        )
         result = CSQ.custom_request_c(
             self._CONFIG_DB,
             query,
@@ -1302,4 +1317,4 @@ class Config(metaclass=ConfigMeta):  # 18.08.25
     user_config: User_config = User_config(common_config=project)  # 18.07.25
     app_args: dict = F.parse_args(sys.argv)
     place: Place = Place(dev=user_config.is_developer)
-    window_manager:WindowManager=WindowManager()
\ No newline at end of file
+    window_manager:WindowManager=WindowManager()
